lib/std/sync/atomic.zig

```diff
@@ -0,0 +1,604 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2015-2021 Zig Contributors
+// This file is part of [zig](https://ziglang.org/), which is MIT licensed.
+// The MIT license requires this copyright notice to be included in all copies
+// and substantial portions of the software.
+
+const std = @import("../std.zig");
+const target = std.Target.current;
+const testing = std.testing;
+
+const AtomicOrder = std.builtin.AtomicOrder;
+const AtomicRmwOp = std.builtin.AtomicRmwOp;
+const AtomicBitRmwOp = enum { Set, Reset, Toggle };
+
+pub fn spinLoopHint() void {
```

Comment:
There's already a copy of this function in the stdlib with several enhancements.

Reply:
Just saw the enhanced version. Will move `std.Thread.spinLoopHint()` here and have it alias this instead